sktime/detection/ggs.py

```diff
@@ -550,12 +531,11 @@ def get_test_params(cls, parameter_set="default"):
         Parameters
         ----------
         parameter_set : str, default="default"
-            Name of the set of test parameters to return, for use in tests. If no
-            special parameters are defined for a value, will return ``"default"`` set.
+            Name of the set of test parameters to return.
 
         Returns
         -------
         params : dict or list of dict
         """
-        params = {"k_max": 10, "lamb": 1.0}
-        return params
+        if parameter_set == "default":
+            return cls(k_max=10, lamb=1.0)
```

Comment:
Given that `get_test_params` has the `@classmethod` decorator you are right to think it should return an instance of the class not a dict but this seems to be causing the CI/CD pipeline to fail.

@fkiraly, do know why `get_test_params` is a classmethod and not a staticmethod?

For reference, I believe this is where the `get_test_params` method is defined in skbase https://github.com/sktime/skbase/blob/9df442ad84e6f5f5a96e4430e3983fd3eef980e6/skbase/base/_base.py#L787.

Reply:
@Alex-JG3  
Well we could convert the `@classmethid` to the `staticmethod` but that might not be useful since every we need to call the function, we would need to pass the argument.